brave/browser-laptop
# Add an option for "no delay" in tab preview timing

closes #8860

this is a follow-up of #9887 and adds a _none_ option for tab preview delay.

Test Plan:

1. Go to pref->tabs
2. Confirm tab preview defaults to "short" 
3. Change tab preview timing to "none"
4. tab previews should be instant
